composites/AlgoliaSearch/AlgoliaSearcher.js

```diff
@@ -83,6 +83,11 @@ class AlgoliaSearcher extends React.Component {
 		if ( searchString === "" ) {
 			return;
 		}
+		let usedQueries = this.addUsedQuery( searchString );
```

Comment:
since now we're using a variable, wouldn't be cleaner to use `usedQueries` below in the setState at line 95 instead of calling `this.addUsedQuery()` again?